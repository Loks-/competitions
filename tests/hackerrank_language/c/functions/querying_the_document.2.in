2
In a prologue a young couple at a diner discuss their life of crime as robbers and the prospect of robbing a restaurant.They decide to rob the diner and begin the holdup.
Hitmen Jules Winnfield and Vincent Vega arrive at an apartment to retrieve a briefcase for their boss.After Vincent checks the contents of the briefcase Jules shoots one of the associates of Brett.
3
3 4 1 2
2 1 1
23
1 1
2
23
10
