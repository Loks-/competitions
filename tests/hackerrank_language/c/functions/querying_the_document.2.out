and
In a prologue a young couple at a diner discuss their life of crime as robbers and the prospect of robbing a restaurant
In a prologue a young couple at a diner discuss their life of crime as robbers and the prospect of robbing a restaurant.They decide to rob the diner and begin the holdup.
